.github/workflows/ci.yml

```
@@ -10,14 +10,25 @@ on:
 
 env:
   CARGO_TERM_COLOR: always
+  # Make sure CI fails on all warnings, including Clippy lints
+  RUSTFLAGS: "-Dwarnings"
 
 jobs:
+  clippy_check:
```

```suggestion
  clippy:
```

Just `clippy` is fine.